artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3476,6 +3514,21 @@
       </xsd:restriction>
    </xsd:simpleType>
 
+   <xsd:simpleType name="dla-routing-type">
+      <xsd:restriction base="xsd:string">
+         <xsd:enumeration value="ANYCAST"/>
+         <xsd:enumeration value="MULTICAST"/>
+         <xsd:enumeration value="AS_ORIGIN">
```

Comment:
Can be removed and use null/not set to mean to use the parent which should imo be the default if not set

Reply:
DLA specific routing type removed.